packages/block-library/src/group/editor.scss

```diff
@@ -39,12 +39,12 @@
 	&::after {
 		content: "";
 		display: flex;
-		border: $border-width dashed currentColor;
-		opacity: 0.4;
-		border-radius: $radius-block-ui;
 		flex: 1 0 $grid-unit-60;
 		pointer-events: none;
 		min-height: $grid-unit-60 - $border-width - $border-width;
+
+		// Dashed placeholder style.
```

Comment:
It's a real nitpick, but do we really need the comment? If it's a general codestyle thing then fine, but otherwise it's kind of duplicative considering it includes the exact same words as the mixin name :D

Reply:
Nitpicks like these are my favorite 👌